app/api/demographics/route.ts

```diff
@@ -68,7 +68,7 @@ return await prisma.demographics.delete({
 }
 
 //export the functions
-export default {
+module.exports = {
```

Comment:
You don't need to worry about this but just so you know, `export default` was causing a Vercel build error here so I just updated it to be `module.exports`